templates/.travis.yml

```diff
@@ -114,12 +114,53 @@ jobs:
+(??)                     env: [ "B2_TOOLSET=gcc-4.8",     "B2_CXXSTD=03,11"    ], addons: *gcc-48    }
+(??)    - { os: "linux", dist: "trusty",  # xenial has libstdc++ from gcc 5.4.0 with newer ABI
+(??)                     env: [ "B2_TOOLSET=gcc-4.9",     "B2_CXXSTD=03,11"    ], addons: *gcc-49    }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=gcc-5",       "B2_CXXSTD=03,11"    ], addons: *gcc-5     }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=gcc-6",       "B2_CXXSTD=11,14"    ], addons: *gcc-6     }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=gcc-7",       "B2_CXXSTD=14,17"    ], addons: *gcc-7     }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=gcc-8",       "B2_CXXSTD=17,2a"    ], addons: *gcc-8     }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=gcc-9",       "B2_CXXSTD=17,2a"    ], addons: *gcc-9     }
+(??)    - { os: "linux", dist: "trusty",  # xenial has libstdc++ from gcc 5.4.0 with newer ABI
+(??)                     env: [ "B2_TOOLSET=clang-3.8",   "B2_CXXSTD=03,11"    ], addons: *clang-38  }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=clang-4.0",   "B2_CXXSTD=11,14"    ], addons: *clang-4   }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=clang-5.0",   "B2_CXXSTD=11,14"    ], addons: *clang-5   }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=clang-6.0",   "B2_CXXSTD=14,17"    ], addons: *clang-6   }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=clang-7",     "B2_CXXSTD=17,2a"    ], addons: *clang-7   }
+(??)    - { os: "linux", env: [ "B2_TOOLSET=clang-8",     "B2_CXXSTD=17,2a"    ], addons: *clang-8   }
+(??)
+(??)    # libc++
+(??)    - { os: "linux", env: [ "B2_TOOLSET=clang-6.0",   "B2_CXXSTD=03,11,14",
+(??)                            "B2_CXXFLAGS=-stdlib=libc++"                   ], addons: *clang-6   }
+(??)    - { os: "osx"  , env: [ "B2_TOOLSET=clang",       "B2_CXXSTD=03,11,17" ]                     }
+(??)
+(??)    # to enable Intel ICC define INTEL_ICC_SERIAL_NUMBER and the following (under development):
+(??)    # - { os: "linux", env: [ "B2_TOOLSET=intel-linux", "B2_CXXSTD=11,14,17" ], addons: *gcc-7,
+(??)    #     script: cd $BOOST_ROOT/libs/$SELF && ci/travis/intelicc.sh                               }
+(??)
+(??)    # uncomment to enable a big-endian build job, just note that it is 5-10 times slower
+(??)    # than a regular build and travis has a 50 minute time limit per job
+(??)    # - os: linux
+(??)    #   env:
+(??)    #     - COMMENT=big-endian
+(??)    #     - B2_CXXSTD=03
+(??)    #     - B2_TOOLSET=gcc
+(??)    #     - B2_DEFINES="define=BOOST_NO_STRESS_TEST=1"
+(??)    #     - BDDE_OS=red
+(??)    #     - BDDE_ARCH=ppc64
+(??)    #   script:
+(??)    #     - cd $BOOST_ROOT/libs/$SELF
+(??)    #     - ci/travis/bdde.sh
+(??)
     - os: linux
       env:
         - COMMENT=codecov.io
         - B2_CXXSTD=03,11
         - B2_TOOLSET=gcc-8
-        - B2_DEFINES="define=BOOST_NO_STRESS_TEST=1"
+        - B2_DEFINES="BOOST_NO_STRESS_TEST=1"
```

Comment:
What if there are multiple `define`-s?

Previously, I think, we could do `B2_DEFINES="define=A=1 define=B=1 define=C=1"

Reply:
Ah yes :/ Reverting

Reply:
A wait. You can still do `B2_DEFINES="A=1 define=B=1 define=C=1"` but the 1-define case is nice and short

Reply:
Sure, works for me, but perhaps a word of comment with multi-define example would be helpful for those new to boost-ci

Reply:
There is one in appveyor as an example. Where to add that comment? I mean: Usually devs can just copy this

Reply:
I see there is example `B2_DEFINES: _POSIX_C_SOURCE=200112L define=__USE_ISOC99` , then that should do I think. Those who want to tweak, have to apply "Read the code, Luke!" ;)